.github/workflows/release.yml

```diff
@@ -0,0 +1,44 @@
+name: release
+
+on:
+  release: # Docs: <https://help.github.com/en/articles/events-that-trigger-workflows#release-event-release>
+    types: [published]
+
+jobs:
+  docker-image:
+    name: Build the docker image
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: docker/setup-qemu-action@v2
+
+      - uses: docker/setup-buildx-action@v2
+
+      # uncomment for publishing on hub.docker.com (don't forget to fillup the repository secrets)
+      #- uses: docker/login-action@v2
+      #  with:
+      #    username: ${{ secrets.DOCKER_LOGIN }}
+      #    password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
```

Comment:
```suggestion
```

This is the correct registry for your `secrets.DOCKER_USERNAME`? If you want to use the default `hub.docker.com` - remove this line

Reply:
@vanhauser-thc All is ok except this one